docs/installation.rst

```diff
@@ -35,21 +35,40 @@ To work with the latest version `on GitHub`_: clone the repository and `cd` into
 its root directory. To install using symbolic links (stay up to date with your
 cloned version after you update with `git pull`) call::
 
-    pip install -e .
+    flit install -s --deps=develop  # from an activated venv or conda env
+    # or
+    flit install -s --deps=develop --python path/to/venv/bin/python
 
-If you intend to do development work, there are some extra dependencies you'll want.
-These can be install with `scanpy` via::
+.. _on GitHub: https://github.com/theislab/scanpy
 
-    pip install -e ".[dev,doc,test]"
+.. note::
 
-.. _on GitHub: https://github.com/theislab/scanpy
+   Due to a `bug in pip`_, using pip’s editable install is more robust than `flit install -s`.
+   Note that this is currently nonstandard behavior and might no longer work once `setup.py` is gone::
```

Comment:
```suggestion
   Due to a `bug in pip`_, packages installed by `flit` can be uninstalled by normal pip operations.
   For now, you can avoid this by using::
```

I feel like we can be more explicit about what happens, and I don't think "once `setup.py` is gone" is going to make sense to people.